modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1290,6 +1290,79 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "123");
 	}
 
+	@description = "Create more than one Table View and choose between the two when deploying an App"
+	@priority = "5"
+	test DeployAppChoosingBetweenDifferentTableViews {
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "Custom Object Title",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.createFormViewAndTableView(
+			fieldType = "Text",
+			formViewName = "Text Form View",
+			tableViewName = "My Table View 1");
+
+		AppBuilderAdmin.switchTab(tabName = "Form Views");
+
+		AppBuilderAdmin.createFormViewAndTableView(
```

Comment:
Hi Tai, I would like to suggest you to add a Text field + Upload field, so you can add a data using the Table View 1 (only Text field), and then change the app to use Table View 2 and then see the data previously added was kept (as expected).